Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,76 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<int?> GetUserIdAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select IdUser from [User] u where u.Email = @Email;";
+            var queryParams = new { Email = accountName };
+            return await connection.QueryFirstOrDefaultAsync<int?>(query, queryParams);
+        }
+
+        public async Task UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesSettings)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+
+            const string updateQuery =
+                @"UPDATE [UserShippingLimit] SET Active = @Active, Interval = @IntervalInDays, Amount = @EmailsAmountByInterval WHERE IdUser = @IdUser;";
+            await connection.ExecuteAsync(updateQuery, contactPoliciesSettings
+            );
+        }
+
+        public async Task<List<ExcludedSubscribersLists>> VerifyUserSubscriberListExist(int? idUser,
```

Comment:
What do you think about return a `IEnumerable` instead of a `List` ?
```suggestion
        public async Task<IEnumerable<ExcludedSubscribersLists>> VerifyUserSubscriberListExist(int? idUser,
```

Reply:
yea, that can be

Reply:
Es tan importante verificar esto?

Reply:
Que ventaja tiene usar enumerable en lugar de list?

Reply:
La ventaja que le encuentro es que no tenemos que convertir la salida de `ReadAsync` a una `List`